cobigen/cobigen-core-systemtest/src/test/java/com/devonfw/cobigen/systemtest/TemplateProcessingTest.java

```diff
@@ -73,35 +73,36 @@ public void cleanup() {
   }
 
   /**
+   * Tests if template sets can be extracted properly
+   *
    * @throws IOException if an Exception occurs
    */
-  @Test
   public void extractTemplateSetsTest() throws IOException {
 
     FileUtils.copyDirectory(new File(testFileRootPath + "templates"),
-        this.cobiGenHome.toPath().resolve("template-sets/downloaded").toFile());
+        this.cobiGenHome.resolve("template-sets/downloaded").toFile());
     CobiGenFactory.extractTemplates();
-    Path adaptedFolder = this.cobiGenHome.toPath().resolve(ConfigurationConstants.CONFIG_PROPERTY_TEMPLATE_SETS_PATH)
+    Path adaptedFolder = this.cobiGenHome.resolve(ConfigurationConstants.CONFIG_PROPERTY_TEMPLATE_SETS_PATH)
         .resolve(ConfigurationConstants.ADAPTED_FOLDER);
     Path extractedJar1 = adaptedFolder.resolve("template-test1-0.0.1");
     Path extractedJar2 = adaptedFolder.resolve("template-test2-0.0.1");
-    assertThat(extractedJar1.toFile()).exists().isDirectory();
-    assertThat(extractedJar2.toFile()).exists().isDirectory();
+    assertThat(Files.isDirectory(extractedJar1));
+    assertThat(Files.isDirectory(extractedJar2));
   }
 
   /**
+   * Test of extract templates with old CobiGen_Templates project existing
+   *
    * @throws IOException if an Exception occurs
    */
-  @Test
   public void extractTemplatesWithOldConfiguration() throws IOException {
 
-    Path cobigenTemplatesParent = this.cobiGenHome.toPath()
-        .resolve(ConfigurationConstants.CONFIG_PROPERTY_TEMPLATES_PATH);
+    Path cobigenTemplatesParent = this.cobiGenHome.resolve(ConfigurationConstants.CONFIG_PROPERTY_TEMPLATES_PATH);
     Files.createDirectories(cobigenTemplatesParent);
     Path cobigenTemplatesProject = cobigenTemplatesParent.resolve(ConfigurationConstants.COBIGEN_TEMPLATES);
     Files.createDirectories(cobigenTemplatesProject);
     CobiGenFactory.extractTemplates();
-    assertThat(cobigenTemplatesProject.toFile()).exists().isDirectory();
+    assertThat(Files.isDirectory(cobigenTemplatesProject));
```

Comment:
again it's better to put isDirectory() at the end

Reply:
Adjusted.